app/src/main/java/net/aiscope/gdd_app/ui/CaptureFlow.kt

```diff
@@ -26,8 +26,9 @@ fun <T> T.attachCaptureFlowToolbar(toolbar: androidx.appcompat.widget.Toolbar)
     }
 }
 
-fun <T> T.goToHome() where T : AppCompatActivity, T : CaptureFlow {
+fun <T> T.goToHome(sampleSaved: Boolean = false) where T : AppCompatActivity, T : CaptureFlow {
```

Comment:
instead to pass a param is possible to have 2 functions with different behaviour? or is better this solution? is possible that in the future, you need to add more params?? also you can save the param in preferences and recover in the other activity, can be this a best solution?

Reply:
A separate function has been created instead of using a parameter.

Regarding the comment about saving the param in preferences, it sounds strange to me to save in preferences something that belongs to the flow.